DiscordBot/index.js

```diff
@@ -1,22 +1,39 @@
 const Discord = require('discord.js');
+const commando = require('discord.js-commando');
 const login = require('./loginCode');
-const bot = new Discord.Client();
+const client = new commando.Client({
+    commandPrefix: '?'
+});
+
+client.registry
+.registerGroups([
+    ['random', 'Random'],
+    ['mud', 'MUD']
+])
+.registerDefaults()
+.registerCommandsIn(__dirname + "/commands");
 
-bot.login(login.key);
+client.login(login.key);
 
-bot.on('message', (message) => {
+console.log("Bot is running");
+
+// below this line are the pre-existing methods that the bot used to parse
```

Comment:
we don't need these anymore

Reply:
Deleted most of them. I commented out the emoji one just in case we want to try to reincorporate that again in the future.

Reply:
sounds beuno :)